src/Engine.ts

```diff
@@ -84,6 +84,10 @@ class Engine {
     return !!this.bucketer.bucket(key, [allocation]);
   }
 
+  getFeature(featureId: string): Feature {
```

Comment:
What about `getFeatureData` ?

Reply:
I didn't want to restrict what to return as someone could be interested in retrieving the audience as well for example